dynamic_example/modules/custom/dynamic_example_homepage/plugins/content_types/cards/cards_item.tpl.php

```diff
@@ -2,15 +2,30 @@
 
 /**
  * @file
- * Companies item.
+ * Cards item.
  */
 ?>
 
-<div class="item">
-  <div class="ui tiny image">
+<div class="ui card">
+  <div class="content">
+    <div class="right floated meta">14h</div>
+    <img class="ui avatar image" src="<?php print $image; ?>"><?php print $title; ?>
+  </div>
+  <div class="image">
     <img src="<?php print $image; ?>">
   </div>
-  <div class="middle aligned content">
-    <a href="<?php print $link; ?>" class="header"><?php print $title; ?></a>
+  <div class="content">
+      <span class="right floated">
+        <i class="heart outline like icon"></i>
+        17 likes
+      </span>
+    <i class="comment icon"></i>
+    3 comments
```

Comment:
Awesome. let's make it event nicer and show the real comments count. if 0 comments, show "no comments", otherwise show the comment count (e.g. "3 comments"). 

Hint, use [format_plural](https://api.drupal.org/api/drupal/includes%21common.inc/function/format_plural/7.x)